packages/oui-message/src/message.html

```diff
@@ -1,14 +1,13 @@
-<div data-ng-if="!$ctrl.dismissed" data-ng-class="'oui-message oui-message_' + $ctrl.type" role="alert">
-  <i class="oui-icon oui-icon_bicolor" data-ng-class="'oui-icon-' + $ctrl.type + '_circle'" aria-hidden="true"></i>
+<div ng-if="!$ctrl.dismissed" ng-class=":: 'oui-message oui-message_' + $ctrl.type" role="alert">
+  <i class="oui-icon oui-icon_bicolor" ng-class=":: 'oui-icon-' + $ctrl.type + '_circle'" aria-hidden="true"></i>
 
-  <div class="oui-message__body" data-ng-class="'oui-message__body_' + $ctrl.type" ng-transclude>
-    <span ng-bind="$ctrl.message"></span>
-  </div>
+    <div class="oui-message__body" ng-class=":: 'oui-message__body_' + $ctrl.type" ng-transclude>
+    </div>
 
-  <button data-ng-if="$ctrl.dismissable === true || ($ctrl.type !== 'error' && $ctrl.type !== 'warning')"
-          class="oui-icon oui-icon-close oui-message__close-button"
-          type="button"
-          data-ng-click="$ctrl.dismiss()"
-          aria-label="Close">
-  </button>
+    <button ng-if=":: $ctrl.type === 'info'"
+      class="oui-icon oui-icon-close oui-message__close-button"
+      type="button"
+      ng-click="$ctrl.dismiss()"
+      aria-label="{{::$ctrl.ariaCloseButtonLabel}}">
```

Comment:
`ng-attr-aria-label` otherwise the accessibility tool could read the version not parsed by angular.

Reply:
done next push